app/js/components/validation.js

```diff
@@ -127,7 +127,7 @@ function validateUrl(value) {
 }
 
 function validateUrn(value) {
-    var regExp = /^urn:[a-z0-9][a-z0-9-]{0,31}:[a-z0-9()+,\-.:=@;$_!*'%/?#]+$/i;
+    var regExp = /^urn:[a-z0-9][a-z0-9-]{0,31}:[a-z0-9()+,\-.:=@;$_!*'%\/?#]+$/i;
```

Comment:
This can easily be covered with a Jest unit test right?

Reply:
I've looked into this and didn't want to take the effort to refactor the js code to ts. And because this was just a third party regex which was copied without taking valid escaping into account I don't want to test it now.